src/Utils/api.ts

```diff
@@ -30,15 +30,15 @@ axiosInstance.interceptors.request.use(
       config.headers["x-locale"] = "en";
     }
 
-    if (typeof Storage !== "undefined") {
-      config.headers["tenant-key"] = `${
-        localStorage.getItem("tenant")
-          ? localStorage.getItem("tenant")
-          : "ten_I9TW3ncG"
-      }`;
-    } else {
-      config.headers["tenant-key"] = "ten_I9TW3ncG";
-    }
+    // if (typeof Storage !== "undefined") {
+    //   config.headers["tenant-key"] = `${
+    //     localStorage.getItem("tenant")
+    //       ? localStorage.getItem("tenant")
+    //       : "ten_I9TW3ncG"
+    //   }`;
+    // } else {
+    //   config.headers["tenant-key"] = "ten_I9TW3ncG";
+    // }
```

Comment:
If we do not need this anymore, let's remove this.

Reply:
done

Reply:
Backend can handle this.
The only issues that might arise can be related to caching but I think that's another subject,